DigitalLearningSolutions.Web/ViewModels/Register/RegisterDelegateByCentre/WelcomeEmailViewModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Register.RegisterDelegateByCentre
+{
+    using DigitalLearningSolutions.Web.ControllerHelpers;
+    using DigitalLearningSolutions.Web.Models;
+
+    public class WelcomeEmailViewModel
+    {
+        public WelcomeEmailViewModel() { }
+
+        public WelcomeEmailViewModel(DelegateRegistrationByCentreData data)
+        {
+            ShouldSendEmail = data.ShouldSendEmail;
+            if (ShouldSendEmail)
+            {
+                Day = data.WelcomeEmailDate!.Value.Day;
+                Month = data.WelcomeEmailDate!.Value.Month;
+                Year = data.WelcomeEmailDate!.Value.Year;
+            }
+        }
+
+        public int? Day { get; set; }
+        public int? Month { get; set; }
+        public int? Year { get; set; }
+        public bool ShouldSendEmail { get; set; }
+        public DateValidator.ValidationResult? DateValidationResult { get; set; }
+
+        public void CleanDate()

```

Comment:
This could be confusing as it doesn't clear when ShouldSendEmail is true. Maybe we can change the name of this method to make it clearer? ClearDateIfNoEmailSent or something similar?